examples/dreambooth/train_dreambooth_lora_sdxl.py

```diff
@@ -755,6 +754,12 @@ def main(args):
         else:
             raise ValueError("xformers is not available. Make sure it is installed correctly")
 
+    if args.gradient_checkpointing:
+        unet.enable_gradient_checkpointing()
+        if args.train_text_encoder:
+            text_encoder_one.gradient_checkpointing_enable()
+            text_encoder_two.gradient_checkpointing_enable()
```

Comment:
the pix2pix script uses `text_encoder_1` and _2 which match the actual internal names. can we make them consistent?